Runtime/Scripts/LyumaAv3Runtime.cs

```diff
@@ -1016,8 +1020,194 @@ class PlayableBlendingState : BlendingState {
+			LyumaAv3Runtime runtime;
+			if (!getTopLevelRuntime("VRCAnimatorPlayAudio", animator, out runtime)) {
+				return;
+			}
+			if (runtime.IsMirrorClone && runtime.IsShadowClone) {
+				return;
+			}
+			if (!runtime)
+			{
+				return;
+			}
+
+			int index = runtime.vrcPlayAudios.IndexOf(playAudio);
+			if (index == -1)
+			{
+				runtime.vrcPlayAudios.Add(playAudio);
+				index = runtime.vrcPlayAudios.IndexOf(playAudio);
+			}
+			if (index % 2 == 0)
+			{
+				return;
+			}
+
+			AudioSource audioSource = animator.transform.Find((string)audioSubType.GetField("SourcePath").GetValue(playAudio))?.GetComponent<AudioSource>();
+			if (audioSource == null)
+			{
+				return;
+			}
+
+			if ((bool)audioSubType.GetField("StopOnEnter").GetValue(playAudio))
+			{
+				audioSource.Stop();
+			}
+
+			object alwaysApply = applySettingsType.GetEnumValues().GetValue(0);
+			object applyIfStopped = applySettingsType.GetEnumValues().GetValue(1);
+
+			if (audioSubType.GetField("LoopApplySettings").GetValue(playAudio).ToString() == alwaysApply.ToString() ||
+			    audioSubType.GetField("LoopApplySettings").GetValue(playAudio).ToString() == applyIfStopped.ToString() &&
+			    !audioSource.isPlaying)
+			{
+				audioSource.loop = (bool)audioSubType.GetField("Loop").GetValue(playAudio);
+			}
+
+			if (audioSubType.GetField("PitchApplySettings").GetValue(playAudio).ToString() == alwaysApply.ToString() ||
+			    audioSubType.GetField("PitchApplySettings").GetValue(playAudio).ToString() == applyIfStopped.ToString() &&
+			    !audioSource.isPlaying)
+			{
+				Vector2 pitch = (Vector2)audioSubType.GetField("Pitch").GetValue(playAudio);
+				audioSource.pitch = UnityEngine.Random.Range(pitch.x, pitch.y);
+			}
+			
+			if (audioSubType.GetField("VolumeApplySettings").GetValue(playAudio).ToString() == alwaysApply.ToString() ||
+			    audioSubType.GetField("VolumeApplySettings").GetValue(playAudio).ToString() == applyIfStopped.ToString() &&
+			    !audioSource.isPlaying)
+			{
+				Vector2 volume = (Vector2)audioSubType.GetField("Volume").GetValue(playAudio);
+				audioSource.volume = UnityEngine.Random.Range(volume.x, volume.y);
+			}
+
+			AudioClip[] clips = (AudioClip[])audioSubType.GetField("Clips").GetValue(playAudio);
+			if (clips.Length > 0 && 
+			    (audioSubType.GetField("ClipsApplySettings").GetValue(playAudio).ToString() == alwaysApply.ToString() ||
+			     audioSubType.GetField("ClipsApplySettings").GetValue(playAudio).ToString() == applyIfStopped.ToString() && !audioSource.isPlaying))
+			{
+				AudioClip clip = null;
+				
+				object Random = randomOrderType.GetEnumValues().GetValue(0);
+				object UniqueRandom = randomOrderType.GetEnumValues().GetValue(1);
+				object Roundabout = randomOrderType.GetEnumValues().GetValue(2);
+				object Parameter = randomOrderType.GetEnumValues().GetValue(3);
```

Comment:
I think order of defined values can be changed over time so I think it's better to use `Enum.ToObject(Type, Int)`.
int representation of enum cannot be changed because serialization format relay on int values.

Reply:
Fixed